nixos/modules/services/hardware/fwupd.nix

```diff
@@ -53,7 +53,7 @@ in {
 
       blacklistPlugins = mkOption {
         type = types.listOf types.str;
-        default = [ "test" ];
```

Comment:
I'm seeing these are always in the config https://github.com/fwupd/fwupd/blob/4adec1b7520726bdef486070ae7b44d404f07503/data/daemon.conf#L9.
Should we mention if someone wants to specify to blacklist certain plugins it will likely be `test` and `invaiid` also?

Reply:
Maybe we could move this to `config` and require mkForce if someone wants to override it.

Reply:
Done that and also moved the list into the package expression.